src/dss/LPOsmAbstract.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.5.12;
+
+// https://github.com/makerdao/univ2-lp-oracle
+interface LPOsmAbstract {
```

Comment:
Can interfaces inherit other interfaces? I'm debating the merits of inheriting `OsmAbstract` here to make the differences more apparent. OTOH--maybe too much coupling? Maybe interfaces should only inherit if the contracts themselves inherit from each other?

Reply:
Yes, they can (see MKRAbstract), but we decided to be explicit here so that when flattening you only need to import a single file.